landoapi/models/landing.py

```diff
@@ -42,35 +51,45 @@ class Landing(db.Model):
     request_id = db.Column(db.Integer, unique=True)
     revision_id = db.Column(db.String(30))
     status = db.Column(db.Integer)
+    auth = db.Column(db.String(30))
```

Comment:
I don't think the auth mechanism should be stored in the database.  The [security design doc](https://docs.google.com/document/d/1cRIdpVqYcn0Mle7vyjJePNw5TcFoZ20N42dLVy5xVEk/edit#) says we should consider a shared API key for validating the client doing the update, and that means we'll end up adding a second migration at a later to remove the auth column.  I recommend we split this user story into two parts: one for this patch to perform status updates with no auth, and a second story to have a special API key for a pingback client to use to update statuses.